R/deploymentTarget.R

```diff
@@ -89,14 +89,14 @@ deploymentTargetForApp <- function(appId,
                                    account = NULL,
                                    server = NULL) {
   accountDetails <- findAccount(account, server)
-  application <- getApplication(accountDetails$account, accountDetails$server, appId)
```

Comment:
These changes are separately submitted in https://github.com/rstudio/rsconnect/pull/805